playbooks/roles/edxapp/defaults/main.yml

```
@@ -1430,7 +1435,7 @@ generic_env_config:  &edxapp_generic_env
       <<: *default_generic_cache
       # Tahoe: RED-1961 Adding a random prefix to invalidate deployment on each deployment to avoid collision.
       #        9100100 is just a random big enough number.
-      KEY_PREFIX: "{{ ansible_hostname|default('staticfiles') }}_{{ 9100100 | random }}_general"
+      KEY_PREFIX: "{{ EDXAPP_APPSEMBLER_BUILD_ID }}_general"
```

I have no idea why it was `_general` but `staticfiles` makes more sense.

```suggestion
      KEY_PREFIX: "{{ EDXAPP_APPSEMBLER_BUILD_ID }}_staticfiles"
```